crypt/aes.go

```diff
@@ -0,0 +1,84 @@
+package crypt
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"io"
+
+	"github.com/goravel/framework/facades"
+)
+
+type AES struct {
+	key []byte
+}
+
+// NewAES returns a new AES hasher.
+func NewAES() *AES {
+	key := facades.Config.GetString("app.key")
+	// check key length before using it
+	if len(key) != 16 && len(key) != 24 && len(key) != 32 {
+		panic("[crypt] app.key length must be 16, 24 or 32")
```

Comment:
我们一般不 panic，而是返回 error，这里我们可以像 `Cache` 模块一样优化为：
```suggestion
		color.Redln("[Crypt] Empty or invalid APP_KEY, please reset it.\nRun command:\ngo run . artisan key:generate")
		return nil
```